lib/firehose/rack/consumer.rb

```diff
@@ -6,6 +6,20 @@ module Rack
     # binds that to the Firehose::Server::Subscription class, which is bound to a channel that
     # gets published to.
     class Consumer
+      class BasicHandler
+        def last_message_sequence(subscription)
+          # this is for backwards compatibility
+          # prefer consistent :last_message_sequence from now on
+          lms = subscription[:last_message_sequence] || subscription[:message_sequence]
```

Comment:
Should there be a note about removing this in the next major version bump? Perhaps we should log a deprecation warning if `subscription[:message_sequence]` is used.